JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/query/RemoteStoreClient.java

```diff
@@ -892,4 +892,93 @@ public void uploadFile(File file) {
             throw new JPSRuntimeException("Upload RDF file failed.", ex);
```

Comment:
Presumably this can be replaced by a call to the override of this function with the extension as an argument.